cmd/release/docs/new_docs/new_docs.go

```diff
@@ -0,0 +1,58 @@
+package new_docs
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+// GenerateNewDocs writes to each doc in provided docs.
+func GenerateNewDocs(newDocsInput []NewDocInput, newDocsDirectory string) ([]string, error) {
+	if err := os.Mkdir(newDocsDirectory, 0777); err != nil {
```

Comment:
you can use os.ModePerm here instead of the octal `0777` if you wish, check out the consts available in os here 
https://pkg.go.dev/os#pkg-constants